frontend/src/components/Job.vue

```diff
@@ -106,4 +124,17 @@ a {
   );
 }
 
+.hide_button {
+  background: none;
+  border: none;
+  cursor: pointer;
+  text-align: right;
+}
+
+.hide_button svg {
+  width: 20px;
+  height: 20px;
+  fill: white;
+}
```

Comment:
svg styles can be moved to **HideIcon** component

Reply:
Solved. I also combined HideIcon and ShowIcon into one component, so that they share this style.